### PR TITLE
Implement fallback from file

### LINKED ISSUE #38: Load CURIE prefixes from file
We need to align with this file: https://biolink.github.io/biolink-model/context.jsonld

We can retrieve at startup, or just maintain a local copy that we keep updated. We may need to supplement with prefixes for terms in our data that aren't represented there.

### PR DESCRIPTION
**Pull request by Shalsh23**
Closes #38

### CODE REVIEW

File: src/main/scala/org/renci/cam/Utilities.scala [hunk 1]
```diff
@@ -27,6 +29,29 @@ object Utilities {
...
+        }
+  } yield curies //getResourcesAsStream
+
+  def getPrefixes: ZIO[HttpClient, Throwable, PrefixesMap] = getBiolinkPrefixesFromURL.orElse(getBiolinkPrefixesFromFile)
+
+  def makePrefixesLayer: UIO[TaskLayer[Has[PrefixesMap]]] = getBiolinkPrefixesFromURL.map(ZLayer.fromManaged())
+
+  val biolinkPrefixes: ZIO[HttpClient, Nothing, TaskLayer[Has[PrefixesMap]]] = ZIO.service
+
+  case class PrefixesMap(prefixesMap: Map[String, String])
```
Comment (Shalsh23) [inline src/main/scala/org/renci/cam/Utilities.scala:55]: @balhoff Does `case class` work, or should I implement a separate class?

Comment (balhoff) [reply src/main/scala/org/renci/cam/Utilities.scala:55]: That works. Make all case classes final though: https://nrinaudo.github.io/scala-best-practices/tricky_behaviours/final_case_classes.html

File: src/main/scala/org/renci/cam/Utilities.scala [hunk 2]
```diff
@@ -27,6 +30,30 @@ object Utilities {
           .collect {
             case (key, Right(value)) => (key, value)
           }
-    } yield curies
+    } yield PrefixesMap(curies)
+
+  def getBiolinkPrefixesFromFile: ZIO[Any, Throwable, PrefixesMap] =
+    for {
+      prefixesStr <- Task.effect(Source.fromFile("prefixes.json")).bracketAuto(source => Task.effect(source.mkString))
```
Comment (balhoff) [inline src/main/scala/org/renci/cam/Utilities.scala:37]: `Source.fromFile("prefixes.json")` won't work once the application is packaged. Use something like `getClass().getResourceAsStream` instead.

File: src/main/scala/org/renci/cam/Utilities.scala [hunk 3]
```diff
@@ -27,6 +30,30 @@ object Utilities {
...
+      prefixesJson = parse(prefixesStr)
+      cursor <- ZIO.fromEither(prefixesJson.map(_.hcursor))
```
Comment (balhoff) [inline src/main/scala/org/renci/cam/Utilities.scala:39]: These lines work fine, but I think they would just make more sense like this:

```scala
prefixesJson <- ZIO.fromEither(parse(prefixesStr))
cursor = prefixesJson.hcursor
```

**Comment by Shalsh23** [comment]
@balhoff Ready for review.

**Review by balhoff — APPROVED** [review]
Looks good.

**Pull request merged**